src/actions/resetPasswordAction.jsx

```diff
@@ -0,0 +1,56 @@
+import swal from "sweetalert2";
+
+export const userPassword = res => ({
+  type: "USER PASSWORD INPUT",
+  payload: res
+});
+
+export const userEmail = res => ({
+  type: "USER EMAIL INPUT",
+  payload: res
+});
+
+export const sendEmailLink = data => dispatch => {
+  /*Sends a pasword reset link*/
+  fetch("http://localhost:8000/api/users/password_reset/", {
```

Comment:
Could you put the url in a dotenv file

Reply:
@shakirandagire , removed localhost url

Reply:
@shakirandagire , if I put the url in env, the mock test won't be able to pick up the fetch function